apps/web/src/utils/poolCandidate.ts

```diff
@@ -596,39 +597,69 @@ export const getQualifiedRecruitmentStatusChip = (
   status: PoolCandidateStatus | null,
   intl: IntlShape,
 ): StatusChipWithDescription => {
-  // placed casual is an exception
-  if (placedAt && status !== PoolCandidateStatus.PlacedCasual) {
-    return {
-      color: "secondary",
-      label: intl.formatMessage(qualifiedRecruitmentStatusLabels.HIRED),
-      description: intl.formatMessage(
-        qualifiedRecruitmentStatusDescriptions.HIRED,
-      ),
-      value: qualifiedRecruitmentStatus.HIRED,
-    };
-  }
+  // there are three possible chips
+  const hiredChip = {
+    color: "secondary",
+    label: intl.formatMessage(qualifiedRecruitmentStatusLabels.HIRED),
+    description: intl.formatMessage(
+      qualifiedRecruitmentStatusDescriptions.HIRED,
+    ),
+    value: qualifiedRecruitmentStatus.HIRED,
+  } as const;
 
-  if (suspendedAt) {
-    return {
-      color: "secondary",
-      label: intl.formatMessage(
-        qualifiedRecruitmentStatusLabels.NOT_INTERESTED,
-      ),
-      description: intl.formatMessage(
-        qualifiedRecruitmentStatusDescriptions.NOT_INTERESTED,
-      ),
-      value: qualifiedRecruitmentStatus.NOT_INTERESTED,
-    };
-  }
+  const notInterestedChip = {
+    color: "secondary",
+    label: intl.formatMessage(qualifiedRecruitmentStatusLabels.NOT_INTERESTED),
+    description: intl.formatMessage(
+      qualifiedRecruitmentStatusDescriptions.NOT_INTERESTED,
+    ),
+    value: qualifiedRecruitmentStatus.NOT_INTERESTED,
+  } as const;
 
-  return {
+  const openToJobsChip = {
     color: "success",
     label: intl.formatMessage(qualifiedRecruitmentStatusLabels.OPEN_TO_JOBS),
     description: intl.formatMessage(
       qualifiedRecruitmentStatusDescriptions.OPEN_TO_JOBS,
     ),
     value: qualifiedRecruitmentStatus.OPEN_TO_JOBS,
-  };
+  } as const;
+
+  if (suspendedAt) {
```

Comment:
I noticed you moved the suspended check ahead of the placed check.  While this makes sense to me, @Jerryescandon can you confirm?  Should the "not interested" chip override the "hired/open to jobs" chip?

Reply:
Perfect, thanks for the clarification!